pom.xml

```diff
@@ -166,8 +166,8 @@
     To change kafka connector dependency, we only need to update this version number config.
     TODO: figure out a way to inject kafka dependency instead of explicitly setting the kafka module dependency -->
     <kafka.version>2.0</kafka.version>
-    <protobuf.version>3.19.2</protobuf.version>
-    <grpc.version>1.41.0</grpc.version>
+    <protobuf.version>3.20.1</protobuf.version>
```

Comment:
The version upgrade is irrelevant. Suggest reverting it in case it has some side effect.

Reply:
+1 on that. Similar to grpc version in the next line.

Reply:
The only reason it's here is that I can't compile it on Apple M1 machine otherwise.
I'm ok reverting it.
Is anyone here has M1 machine? Or maybe I'm missing something obvious.

Reply:
Interesting. We do have people developing using M1 machine. What issue do you run into compiling it?
Let's revert it in this PR, and we may have a separate issue discussing it if you still cannot figure it out.

Reply:
@gpogulsky you might want to clean up the libraries under `~/.m2/repository/` dir, which might be helpful.

Reply:
I'll revert it.
Not going to argue here.
The error is this
[ERROR] Failed to execute goal org.xolstice.maven.plugins:protobuf-maven-plugin:0.6.1:compile-custom (default) on project pinot-common: Unable to resolve artifact: Missing:
[ERROR] ----------
[ERROR] 1) io.grpc:protoc-gen-grpc-java:exe:osx-aarch_64:1.41.0

I'm surprised nobody encountered.